### PR TITLE
feat: added icon on button

### PR DESCRIPTION
**Pull request by GutierrezGo**
References https://autoricardo.atlassian.net/browse/DM-727

## Motivation and context

Primary and Secondary buttons with icon

## Before

No icons on the button component 🧟‍♀️

## After

Icons on the button component 🙌🏼

## How to test

Please check the new button variations (left and right button).

### CODE REVIEW

File: src/components/button/index.tsx [hunk 1]
```diff
@@ -49,11 +51,16 @@ const Button: FC<Props> = ({ children, ...props }) => {
...
     isExternal,
+    leftIcon,
+    rightIcon,
     ...rest
   } = props;
 
   return (
     <ChakraButton
+      leftIcon={leftIcon}
+      rightIcon={rightIcon}
```
Comment (GutierrezGo) [inline src/components/button/index.tsx:62]: I double checked with Enrique and we could have the case for a right icon on the button.

File: src/themes/components/button.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { SystemStyleObject } from '@chakra-ui/styled-system';
 const baseStyle: SystemStyleObject = {
   borderRadius: 'sm',
   px: 'lg',
+  whiteSpace: 'break-spaces',
```
[the remark this answers is not in the document]
Comment (GutierrezGo) [reply src/themes/components/button.ts:7]: The text on a button shouldn't be a story. The use case should be two lines at most. Besides the designers try to adapt a good text for the buttons.

The high on the buttons are not dynamic. We have fix highs `36px` and `48px`.

**Review by Marine-Berthier — APPROVED** [review]

**Review by talamcol — APPROVED** [review]
LGTM

**Review by boyarskiy — APPROVED** [review]
👍

**Pull request merged**